src/libs/ReportUtils.ts

```diff
@@ -1193,7 +1193,7 @@ function hasOnlyTransactionsWithPendingRoutes(iouReportID: string | undefined):
  * If the report is a thread and has a chat type set, it is a workspace chat.
  */
 function isWorkspaceThread(report: OnyxEntry<Report>): boolean {
-    return isThread(report) && isChatReport(report) && !!getChatType(report);
+    return isThread(report) && isChatReport(report) && !isGroupChat(report) && !!getChatType(report);
 }
```

Comment:
Not tested but I think we may have same bug with self dm reports. Similar bugs may occur in the future, it's safer to write this as:
```js
    return isThread(report) && isChatReport(report) && [<LIST OF WORKSPACE CHAT TYPES>].includes(getChatType(report));
```

Reply:
Checking... this one got me because `group` is a `chatType` but `chatType` has mostly been used for "workspace" type things until now.

Reply:
Your idea to have these be constants is a good one 👍

Reply:
Track expense is still behind a beta in any case. But good catch here. Made the change. Thanks!